en/includes/guides/authentication/passwordless-login/add-passwordless-login-with-passkey.md

```diff
@@ -151,3 +152,103 @@ Follow the steps below to use an enrolled passkey to sign in to an application.
 
 !!! note
     During passkey progressive enrollment, if a user wishes to use a federated authenticator, they should have their external accounts already provisioned within {{product_name}}. If, for example, a user logs in with Google using an account not provisioned in {{product_name}}, passkey enrolment results in an error and the login flow fails.
+
+
+## Make application a FIDO trusted app
+
+If you are working with a mobile application that implements passkeys, making it a FIDO trusted app allows the hosting platform (iOS or Android) to validate the authenticity of the application by calling the corresponding validation service. This ensures that the authentication requests originate from a legitimate application, safeguarding against malicious attempts to steal credentials.
```

Comment:
```suggestion
When using app-native authentication with a mobile application that implements passkeys, the hosting platform (iOS or Android) performs a validation to ensure the authenticity of the application. For this validation to succeed, the application must be designated as a FIDO trusted app within {{product_name}}. This ensures that the authentication requests originate from a legitimate application, safeguarding against malicious attempts to steal credentials.
```

Reply:
This section is required only if we are configuring app-native authentication. This configuration is not needed if we are using browser based authentication.